Wikipedia/Code/AboutViewController.m

```diff
@@ -59,6 +59,15 @@ - (void)wmf_setTextFontSize {
     [self evaluateJavaScript:fontSizeJS completionHandler:nil];
 }
 
+- (void)wmf_setTextFontColor:(WMFTheme *)theme {
+    NSString *fontColorJS = [NSString stringWithFormat:@"document.body.style.color = '#%@';\
+                             document.styleSheets[0].rules[2].style.color = '#%@';\
```

Comment:
I used the inspector and was able to access the rules from style id handle. 
`document.getElementById(#ID_HERE#).sheet.rules`

Was also able to get the styles by selector name. Ping me if you want a quick pair session.